setup.cfg

```diff
@@ -1,2 +1,10 @@
 [wheel]
 universal = 1
+
+[isort]
```

Comment:
What is going on here and what does it have to do with the directory map interface?